homework/myapp/admin.py

```diff
@@ -0,0 +1,83 @@
+from django.contrib import admin
+from .models import Customer, Product, Order
+
+
+@admin.action(description="Сбросить количество продукта в ноль")
+def reset_product_quantity(modeladmin, request, queryset):
+    queryset.update(product_quantity=0)
+
+
+class CustomerAdmin(admin.ModelAdmin):
```

Comment:
Можно использовать декоратор `admin.register` для регистрации костюмной модели 
```suggestion
@admin.register(Customer)
class CustomerAdmin(admin.ModelAdmin):
```